### PR TITLE
solving the median of two sorted arrays problem (day 10, first point)…

### PR DESCRIPTION
**Pull request by CHAriina**
…. Changing namespaces in tests to improve the structure

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstring.Tests.cs [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ﻿using LeetCodeTraining.LongestSubstringWithoutRepeating3;
-
-namespace LeetCodeTraining.Tests
+namespace LongestSubstringWithoutRepeatingTests
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstring.Tests.cs]: fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

File: ScnSoft-Education/LeetCodeTraining.Tests/MedianOfTwoSortedArrays.Tests.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿using LeetCodeTraining.MedianOfTwoSortedArrays6;
+namespace MedianOfTwoSortedArraysTests
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/MedianOfTwoSortedArrays.Tests.cs]: fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

File: ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿using LeetCodeTraining.StringToInteger8;
-namespace LeetCodeTraining.Tests
+namespace StringToIntegerTests
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs]: fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

File: ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs [hunk 4]
```diff
@@ -1,6 +1,5 @@
 ﻿using LeetCodeTraining.TwoSum1;
-
-namespace LeetCodeTraining.Tests
+namespace TwoSumTests
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs]: fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

File: ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays6/MedianOfTwoSortedArrays.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+{
+    public class Solution
+    {
+        public double FindMedianSortedArrays(int[] nums1, int[] nums2)
+        {
+            int m = nums1.Length;
+            int n = nums2.Length;
+            int[] merged = new int[m + n];
+
+            int i = 0, j = 0, k = 0;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays6/MedianOfTwoSortedArrays.cs]: use real name for variables instead magic names

File: ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstring.Tests.cs (hunk 1 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstring.Tests.cs]: fixed errors in namespace

File: ScnSoft-Education/LeetCodeTraining.Tests/MedianOfTwoSortedArrays.Tests.cs (hunk 2 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/MedianOfTwoSortedArrays.Tests.cs]: fixed errors in namespace

File: ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs (hunk 3 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/StringToInteger8.Tests.cs]: fixed errors in namespace

File: ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs (hunk 4 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs]: fixed errors in namespace

File: ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays6/MedianOfTwoSortedArrays.cs (hunk 5 above)
Comment (CHAriina) [reply ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays6/MedianOfTwoSortedArrays.cs]: fixed variable names

**Pull request merged**